analyzer.go

```diff
@@ -153,12 +156,48 @@ func (gosec *Analyzer) Process(buildTags []string, packagePaths ...string) error
 		Tests:      gosec.tests,
 	}
 
-	for _, pkgPath := range packagePaths {
-		pkgs, err := gosec.load(pkgPath, config)
-		if err != nil {
-			gosec.AppendError(pkgPath, err)
+	type result struct {
+		pkgPath string
+		pkgs    []*packages.Package
+		err     error
+	}
+
+	results := make(chan result)
+	jobs := make(chan string)
+
+	var wg sync.WaitGroup
+	wg.Add(gosec.concurrency)
```

Comment:
Normally the wait group is incremented in the loop where the workers are started.